### DISCUSSION TITLE
How to get a list of multiple query parameters (/?status=1&status=3) in query Schema?

### DISCUSSION THREAD

**Discussion by zhibeen** [apiflask/apiflask#168] [Q&A] [answered]
InSchema 如何定义 可以支持 这种格式的查询
/drawbacks?status[]=0&status[]=1

**Comment by greyli** [comment, marked as answer]
```
from apiflask.fields import String, List

class QuerySchema(Schema):
    status = List(String())
```

下次中文问题请发到 [HelloFlask 论坛](https://discuss.helloflask.com)。

**Reply by zhibeen** [reply]
如果query parameters (/?status[]=1&status[]=3)  这种情况该如何处理

**Reply by greyli** [reply]
```python
status = List(String(data_key='status[]'))
```
没测试，可以试一下。

**Reply by zhibeen** [reply]
测试了 带有 [ 和 ] 这两个特殊字符不可以

**Reply by greyli** [reply]
那可以试试给 Schema 用 pre_load 装饰器注册一个方法，然后手动从 request.args.getlist('status[]') 获取。
